tools/README.adoc

```diff
@@ -0,0 +1,133 @@
+[%hardbreaks]
+:toc: left
+:toclevels: 3
+
+
+== Overview
+
+This folder contains the following tools:
+
+. `sgcollect_info` -- a tool that collects diagnostics from Sync Gateway, such as logs, heap profile, cpu profile, expvars, etc.
+. A copy of `cbcollect_info` and `tasks.py`, which was refactored from the upstream `cbollect_info` in the [ns_server repository](https://github.com/couchbase/ns_server/blob/master/cbcollect_info), which has since diverged from this version.  `sgcollect_info` depends on the `cbcollect_info` in this repo.
```

Comment:
Looks like AsciiDoc links should be in the form of `URL[link text]` to render properly.

Reply:
Old habits die hard ;-)